src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectLaunchTargetsProvider.cs

```diff
@@ -630,4 +630,24 @@ private enum StringState
             NormalCharacter, EscapedCharacter, QuotedString, QuotedStringEscapedCharacter
         }
     }
+
+    [Serializable]
```

Comment:
Does this need to be serializable? If so, it should have the serialization constructor too. I suspect it doesn't though.

Reply:
I removed, it was added by codefix.